.azuredevops/modulePipelines/ms.dataprotection.backupvaults.yml

```diff
@@ -0,0 +1,55 @@
+name: "DataProtection - BackupVaults"
+
+parameters:
+  - name: removeDeployment
+    displayName: Remove deployed module
+    type: boolean
+    default: true
+  - name: prerelease
+    displayName: Publish prerelease module
+    type: boolean
+    default: false
+
+pr: none
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - "/.azuredevops/modulePipelines/ms.dataprotection.backupvaults.yml"
+      - "/.azuredevops/pipelineTemplates/module.*.yml"
+      - "/arm/Microsoft.DataProtection/vaults/*"
+      - "/arm/.global/global.module.tests.ps1"
+    exclude:
+      - "/**/*.md"
+
+variables:
+  - template: "../../global.variables.yml"
+  - group: "PLATFORM_VARIABLES"
+  - name: modulePath
+    value: "/arm/Microsoft.DataProtection/backupVaults"
+
+stages:
+  - stage: Validation
+    displayName: Static validation
+    jobs:
+      - template: /.azuredevops/pipelineTemplates/jobs.validateModulePester.yml
+
+  - stage: Deployment
+    displayName: Deployment validation
+    jobs:
+      - template: /.azuredevops/pipelineTemplates/jobs.validateModuleDeployment.yml
+        parameters:
+          removeDeployment: "${{ parameters.removeDeployment }}"
+          deploymentBlocks:
+            - path: $(modulePath)/.parameters/parameters.json
+              templateFilePath: $(modulePath)/deploy.bicep
```

Comment:
Not required

Reply:
Just the reference :) A default is implemented in the template itself: 
![image](https://user-images.githubusercontent.com/5365358/172400541-ee61eecf-1bf4-472e-be98-a437495d22c4.png)